examples/pytorch/hilander/PSS/Smooth_AP/src/losses.py

```diff
@@ -0,0 +1,357 @@
+# repo originally forked from https://github.com/Confusezius/Deep-Metric-Learning-Baselines
+
+###################### LIBRARIES #################################################
```

Comment:
what's the difference of the loss.py as compared to Smooth_AP_loss.py? or are they duplicates of each other?

Reply:
Would be highly appreciated if you could send out reviews in a single batch, otherwise my inbox is bombed :)